### PR TITLE
Set some meta tags

### PR DESCRIPTION
**Pull request by xPaw**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -4,10 +4,18 @@
 
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, user-scalable=no">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 
 	<title>{% if page.title %}{{ page.title }} · {% endif %}{{ site.title }}</title>
 	<meta name="description" content="{{ site.description }}">
+	<meta name="application-name" content="The Lounge">
```
Comment (astorije) [inline _layouts/default.html]: Not sure we want that one, according to https://stackoverflow.com/a/24723106

File: _layouts/default.html [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
+	<meta name="theme-color" content="#415364">
+	<meta property="twitter:card" content="summary">
```
Comment (astorije) [inline _layouts/default.html]: Is that one enough? Don't we need the other `twitter:*` too?

File: _layouts/default.html [hunk 3]
```diff
@@ -4,10 +4,18 @@
...
+	<meta property="og:type" content="website">
+	<meta property="og:site_name" content="The Lounge">
+	<meta property="og:locale" content="en_US">
+	<meta property="og:title" content="{{ page.title }}">
+	<meta property="og:description" content="{{ site.description }}">
+	<meta property="og:image" content="https://thelounge.chat/img/meta_image.png">
```
Comment (astorije) [inline _layouts/default.html]: I feel like we already have this one somewhere. Can we use one from the `assets` folder or elsewhere?

File: _layouts/default.html (hunk 2 above)
Comment (xPaw) [reply _layouts/default.html]: It falls back to og tags.

File: _layouts/default.html (hunk 1 above)
Comment (xPaw) [reply _layouts/default.html]: Guess should be removed, yeah

**Pull request merged**